src/test-utilities/enzyme.tsx

```diff
@@ -10,10 +10,15 @@ import {DeepPartial} from '../types';
 import {
   createPolarisContext,
   AppProviderContext,
-  Context as AppProviderContextType,
 } from '../components/AppProvider';
 // eslint-disable-next-line shopify/strict-component-boundaries
 import {Provider as FrameProvider, FrameContext} from '../components/Frame';
+// eslint-disable-next-line shopify/strict-component-boundaries
```

Comment:
When we export the return value from `React.createContext` we should be able to import this from `'../components'`